webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -26,3 +30,40 @@ export const addSearchToURL = (path, query) => {
   url.addSearch({ ...query, include_permissions: true });
   return url.toString();
 };
+
+export const getDuration = (start, finish) => {
+  const selectedLocale = documentLocale();
+  if (!start && !finish)
+    return { text: __('N/A'), tooltip: __('No start or end dates') };
+
+  if (!start && finish) {
+    return { text: __('N/A'), tooltip: __('Task was canceled') };
+  }
+
+  const dateOptions = {
+    largest: 1,
+    language: selectedLocale,
+    fallbacks: ['en'],
+  };
+
+  start = new Date(isoCompatibleDate(start)).getTime();
+  if (start && !finish) {
+    finish = new Date().getTime();
```

Comment:
Looks like `getTime` is redundant. I assume this will bring the same results:
```js
start = new Date(isoCompatibleDate(start));
finish = new Date();
duration = finish - start; // should contain the number of the difference
```